graphql/resolvers.go

```diff
@@ -870,6 +870,25 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) Host(ctx context.Context, limit *int, page *int, hostID *string) (*restModel.APIHost, error) {
+	host, err := host.FindOneByIdOrTag(*hostID)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error Fetching host: %s", err.Error()))
+	}
+
+	if host == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find host with id %s", *hostID))
```

Comment:
always include the error in the formatted message

Reply:
I disagree - the error will be a db error message, which the user won't know what to do with

Reply:
yeah that's true. they are very useful for us though when debugging.

Reply:
@john-m-liu I thought about this and the error message should always be included. we can show whatever we want on the front end for the error message, it does not need to be the actual message from the server. if the error object is not included in the server error then it is not included in BugSnag either, which it should be.